pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "moviecenter"
+version = "0.1.0"
+description = ""
+authors = ["Eli"]
+
+[tool.poetry.dependencies]
+python = "3.10"
+pydantic = "1.10.2"
```

Comment:
What about `fastapi`?

Reply:
No tests for the web server yet so didn't have to add it :)

Reply:
That's not just for tests, that's the file specifying all of your app's dependencies.
If one installs your app using that file they won't be able to run your app (in fact, there's no place your repo specifying `fastapi` is needed to run/develop your app)